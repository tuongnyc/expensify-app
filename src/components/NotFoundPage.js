import React from 'react';
import { Link } from 'react-router-dom';

// <Link> is similar to <a>, however it does not do whole page refresh
// so, it is much faster!
const NotFoundPage = () => (
    <div>
    404 -> <Link to="/">Go Home </Link>
    </div>
);


export default NotFoundPage;