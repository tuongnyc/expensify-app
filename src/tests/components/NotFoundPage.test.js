import React from 'react';
import NotFoundPage from '../../components/NotFoundPage';
import { shallow } from 'enzyme'; 
import toJSON from 'enzyme-to-json';

test('should render NotFoundPage correctly', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
})