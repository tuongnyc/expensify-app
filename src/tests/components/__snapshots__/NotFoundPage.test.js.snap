// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage correctly 1`] = `
<div>
  404 -&gt; 
  <Link
    replace={false}
    to="/"
  >
    Go Home 
  </Link>
</div>
`;
